### PR TITLE
Add Search

### PR DESCRIPTION
**Pull request by Joozty**
At the moment the search box is connected to the /works endpoint. This endpoint may be changed in the future but I expect from our point of view it won't be a big change as the interface should be almost identical. 

Changes:
1. search box is connected to the API (/works)
2. search box also show suggestions
3. when user search and API request takes more than 500ms we show loading placeholders

### CODE REVIEW

File: templates/data-provider/search.jsx [hunk 1]
```diff
@@ -44,8 +53,8 @@ const DataProviderOutputsSearch = ({
       className={className}
     >
       {suggestions.map((el) => (
-        <Select.Option key={el.id} id={el.id} value={el.name} icon="#magnify">
-          {el.name}
+        <Select.Option key={el.id} id={el.id} value={el.title} icon="#magnify">
```
Comment (Joozty) [inline templates/data-provider/search.jsx:56]: Not sure if we need suggestions at the moment. Let me know.

Comment (viktor-yakubiv) [reply templates/data-provider/search.jsx:56]: It doesn't look good at the moment. I would rather remove it.

File: templates/data-provider/result-placeholder.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      />
+    </div>
+    <div className={styles.resultContent}>
+      <Placeholder.Heading />
+      <Placeholder.Line />
+      <Placeholder.Spacer />
+      <Placeholder.Line />
+      <Placeholder.Line />
+      <Placeholder.ShortLine />
+    </div>
```
Comment (viktor-yakubiv) [inline templates/data-provider/result-placeholder.jsx:23]: You may find useful oacore/design#349

**Review by viktor-yakubiv — APPROVED** [review]
Looks good to me! 👍

For the search result:
- image should be more narrow — 80px
- it should have an aspect ratio close to A4 (1.44) but I find 1.12 a bit better looking
- image should be aligned to the top
- text snipper should be cut by 2 lines exactly

File: templates/data-provider/result-placeholder.jsx (hunk 2 above)
Comment (Joozty) [reply templates/data-provider/result-placeholder.jsx:23]: I will wait for it.

Comment (viktor-yakubiv) [reply templates/data-provider/result-placeholder.jsx:23]: The idea behind the PR is complete preserving the layout but changing only one flag. I cannot find any other way to test it than moving search result component to design modules and do it there. I can do this.

**Pull request merged**